### PR TITLE
Added CTIX Connector v1.0.0 for XSOAR Marketplace (Partner ID: cyware-238)

### PR DESCRIPTION
**Pull request by gladipero**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CTIX/Integrations/CTIX/CTIX_test.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+SECRET_KEY = "secret_key"
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+def test_ip(requests_mock):
+    from CTIX import Client, ip_details_command
```
Comment (esharf) [inline Packs/CTIX/Integrations/CTIX/CTIX_test.py:17]: unit tests should contain a given-when-then section.
e.g.
Given
\- 6.7.8.9
When
\- collecting data for an existing indicator
Then
\- ensure the return data contains ...

File: Packs/CTIX/Integrations/CTIX/CTIX.py [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+
+        :type enhanced: ``bool``
+        :param enhanced: Enhanced search flag
+
+        :return: dict containing the IP Details as returned from the API
+        :rtype: ``Dict[str, Any]``
+        """
+        ip_string = ",".join(ip)
+
+        if enhanced == "True":
```
Comment (esharf) [inline Packs/CTIX/Integrations/CTIX/CTIX.py:89]:
```suggestion
        if argToBool(enhanced):
```

File: Packs/CTIX/Integrations/CTIX/CTIX.py [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+        :type enhanced: ``bool``
+        :param enhanced: Enhanced search flag
+
+        :return: dict containing the domain details as returned from the API
+        :rtype: ``Dict[str, Any]``
+        """
+
+        domain_string = ",".join(domain)
+
+        if enhanced == "True":
```
Comment (esharf) [inline Packs/CTIX/Integrations/CTIX/CTIX.py:110]:
```suggestion
        if argToBool(enhanced):
```

File: Packs/CTIX/Integrations/CTIX/CTIX.py [hunk 4]
```diff
@@ -0,0 +1,384 @@
...
+        :type enhanced: ``bool``
+        :param enhanced: Enhanced search flag
+
+        :return: dict containing the url details as returned from the API
+        :rtype: ``Dict[str, Any]``
+        """
+
+        url_string = ",".join(url)
+
+        if enhanced == "True":
```
Comment (esharf) [inline Packs/CTIX/Integrations/CTIX/CTIX.py:132]:
```suggestion
        if argToBool(enhanced):
```

File: Packs/CTIX/Integrations/CTIX/CTIX.py [hunk 5]
```diff
@@ -0,0 +1,384 @@
...
+
+        :type enhanced: ``bool``
+        :param enhanced: Enhanced search flag
+
+        :return: dict containing the file details as returned from the API
+        :rtype: ``Dict[str, Any]``
+        """
+
+        file_string = ",".join(file)
+        if enhanced == "True":
```
Comment (esharf) [inline Packs/CTIX/Integrations/CTIX/CTIX.py:153]:
```suggestion
        if argToBool(enhanced):
```

File: Packs/CTIX/Integrations/CTIX/CTIX.yml [hunk 6]
```diff
@@ -0,0 +1,1137 @@
...
+  type: 4
+  required: true
+  additionalinfo: Enter the Access Key from the CTIX application.
+- display: Secret Key
+  name: secret_key
+  defaultvalue: ""
+  type: 4
+  required: true
+  additionalinfo: Enter the Secret Key from the CTIX application.
+script:
```
Comment (esharf) [inline Packs/CTIX/Integrations/CTIX/CTIX.yml:39]: please add the proxy and trust any certificate params. (Required also in the demisto-sdk validates.

**Review by esharf — CHANGES_REQUESTED** [review]
please fix also the errors in the validations and lints.

**Comment by gladipero** [comment]
Closing this PR as will be opening via the corporate account due to CLA issues.

**Comment by bakatzir** [comment]
opened in: https://github.com/demisto/content/pull/10251

**Pull request closed without merging**